test/users.js

```diff
@@ -0,0 +1,67 @@
+const chai = require('chai'),
+  dictum = require('dictum.js'),
+  server = require('./../app'),
+  orm = require('../app/orm'),
+  should = chai.should();
+
+describe('/users POST', () => {
+  it('should fail because the password is too short', (done) => {
+    chai.request(server)
+      .post('/users')
+      .send({
+        firstName: 'xxxxxx', lastName: 'zzzzzzzzz',
+        password: '1234', email: '123456@wolox.com.ar'
+      })
+      .catch((err) => {
+        err.response.should.be.json;
+        err.response.body.should.have.property('error');
```

Comment:
You could also check that the error message is the one we expect in this case `The password is too short`